zerver/tests/test_narrow.py

```diff
@@ -429,6 +429,38 @@ def test_ok_to_include_history(self) -> None:
         ]
         self.assertTrue(ok_to_include_history(narrow, user_profile))
 
+        # Tests for guest user
+        user_profile = self.example_user("polonius")
+        user_profile.is_guest = True
+        user_profile.save(update_fields=['is_guest'])
```

Comment:
I don't think you need these two lines; he's created as a guest user in `populate_db`.

Reply:
Sorry, just added for readability, I've seen some similar instances like this long ago(maybe I've misinterpreted them). Updating.. thanks :)